quilt/tools/store.py

```diff
@@ -221,6 +243,34 @@ def temporary_object_path(self, name):
         """
         return os.path.join(self._path, self.TMP_OBJ_DIR, name)
 
+    def prune(self):
+        """
+        Clean up all objects not referenced by any packages.
+        """
+
+        objdir = os.path.join(self._path, self.OBJ_DIR)
+        all_obj = os.listdir(objdir)
+        # TODO: the temporary object directory probably shouldn't
+        # live inside the objects directory. Remove the hard-coded
+        # 'tmp' below.
+        all_obj.remove('tmp')
+
+        for pkg in self.iterpackages():
+            for node in pkg.get_contents().preorder():
+                # TODO: the or below isn't scalable. Add a common baseclass for
+                # File and Table nodes like DataNode in nodes.py.
+                if isinstance(node, TableNode) or isinstance(node, FileNode):
```

Comment:
python has syntax for multiple checks?
https://stackoverflow.com/questions/33311258/python-check-if-isinstance-any-type-in-list

Reply:
Good call. Fixed.